packages/edit-site/src/components/list/style.scss

```diff
@@ -114,3 +114,23 @@
 		border-top: none;
 	}
 }
+
+.edit-site-list-added-by__icon {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: $grid-unit-40;
+	height: $grid-unit-40;
+	background: $gray-800;
+	border-radius: 100%;
+
+	svg {
+		fill: $white;
+	}
+}
+
+.edit-site-list-added-by__avatar {
+	width: $grid-unit-40;
+	height: $grid-unit-40;
+	border-radius: 100%;
+}
```

Comment:
@jameskoster Would be great to get an early check on these styles. Feel free to push a commit or comment with any adjustments you want to make.

Reply:
Styles looking good so far. I did notice that customised theme templates are displaying strangely, but assume you're still working on that?

<img width="1065" alt="Screenshot 2021-11-23 at 10 24 40" src="https://user-images.githubusercontent.com/846565/143007748-86317178-abfb-4e83-9fa5-b412338de8b8.png">

Reply:
Yeah, I noticed it's a bit glitchy when loading the image too. I'll look at improving it 👍

Reply:
This should be fixed now.